L1Trigger/L1THGCalUtilities/python/clustering2d_v2.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
+from L1Trigger.L1THGCal.hgcalBackEndLayer1Producer_cfi import dummy_C2d_params, \
+                                                              distance_C2d_params, \
+                                                              topological_C2d_params, \
+                                                              constrTopological_C2d_params, \
+                                                              stage1truncation_proc, \
+                                                              truncation_params
+from L1Trigger.L1THGCal.customClustering import set_threshold_params
```

Comment:
Unnecessary imports should be removed as well

Reply:
Sorry I missed that, now removed